pr_agent/tools/pr_reviewer.py

```diff
@@ -103,4 +108,37 @@ def _prepare_pr_review(self) -> str:
 
         if settings.config.verbosity_level >= 2:
             logging.info(f"Markdown response:\n{markdown_text}")
-        return markdown_text
\ No newline at end of file
+        return markdown_text
+
+    def _publish_inline_code_comments(self):
```

Comment:
The '_publish_inline_code_comments' method is quite long and does multiple things. Consider breaking it down into smaller, more manageable methods. This can improve code readability and maintainability. [medium]